deploy/gateway-production.json

```diff
@@ -257,8 +257,7 @@
                             {
                                 "name": "gateway-vmss-nic",
                                 "properties": {
-                                    "primary": true,
```

Comment:
what was this primary parameter used for ?

Reply:
I've put this back